java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -280,6 +280,24 @@ CompletableFuture<Boolean> pexpireAt(
 
     // TODO move ScriptingAndFunctionsBaseCommands
     // TODO add note about routing on cluster client
```

Comment:
Should we move these comments below this function? I'm assuming they apply to invokeScript